src/main/java/nl/hsac/fitnesse/fixture/slim/StringFixture.java

```diff
@@ -33,13 +42,60 @@ public int lengthOf(String value) {
         return length;
     }
 
+    /**
+     * Checks if an input string is empty: null or length() = 0.
+     *
+     * @param value value to check.
+     * @return {@code true} if empty, or {@code false} otherwise.
+     */
+    public boolean isEmpty(String value) { return StringUtils.isEmpty(value); }
+
+    /**
+     * Checks if a string is not empty: not null and length() != 0.
+     *
+     * @param value value to check.
+     * @return {@code true} if filled (not empty), {@code false} otherwise.
+     */
+    public boolean isNotEmpty(String value) { return !isEmpty(value); }
+
+    /**
+     * Checks if a value meets the symbol naming convention ('$xyz').
+     * This indicates an undefined FitNesse symbol.
+     * http://fitnesse.org/FitNesse.SuiteAcceptanceTests.SuiteSlimTests.SlimSymbols.NamingConvention
+     *
+     * @param value value to check.
+     * @return {@code true} if undefined (value is symbol name-like), {@code false} otherwise.
+     */
+    public boolean isUndefined(String value) {
```

Comment:
Having a symbol in your tests that may or may not be assigned a value sounds like a code smell of your tests.

If you worry a symbol may have been assigned a value in a previous test in a suite, just ensure you assign a known (empty, or otherwise not occurring value) in your SetUp.